app/assets/stylesheets/pages/articles/article-list/_pagination.scss

```diff
@@ -0,0 +1,68 @@
+.pagination{
+	margin-top: 64px;
+	margin-bottom: 82px;
+
+	&__list-wrapper{
+		list-style: none;
+		padding-left: 0;
+	}
+
+	&__list-page{
+		display: inline-block;
+		margin-right: 10.2px;
+
+		&:hover .pagination__list-link{
+			background-color: $colorPrimary;
+			color: #fff;
+		}
+
+		&:first-child{
+			margin-left: 0;
```

Comment:
maybe you wanna check it again, did you really need margin left 0 for the first child? because i don't see it needed

Reply:
sorry i forgot to delete this code